chromium_src/chrome/browser/ui/webui/site_settings_helper.cc

```diff
@@ -4,14 +4,15 @@
  * You can obtain one at http://mozilla.org/MPL/2.0/. */
 
 #define HasRegisteredGroupName HasRegisteredGroupName_ChromiumImpl
-#define ContentSettingsTypeFromGroupName \
-  ContentSettingsTypeFromGroupName_ChromiumImpl
 #define ContentSettingsTypeToGroupName \
   ContentSettingsTypeToGroupName_ChromiumImpl
+#define BRAVE_SITE_SETTINGS_HELPER_CONTENT_SETTINGS_TYPE_FROM_GROUP_NAME \
+  ContentSettingsType ContentSettingsTypeFromGroupName_ChromiumImpl(     \
+      const std::string& name) {
 
```

Comment:
This extra blank line will fail lint

Reply:
Yep, sure did :) Added a comment line to pacify lint.